packages/postcss-minify-gradients/src/index.js

```diff
@@ -55,9 +63,14 @@ function optimise(decl) {
           args[0].length === 3
         ) {
           node.nodes = node.nodes.slice(2);
-          node.nodes[0].value = angles[node.nodes[0].value.toLowerCase()];
+          node.nodes[0].value =
+            angles[
+              /** @type {'top'|'right'|'bottom'|'left'}*/ (
+                node.nodes[0].value.toLowerCase()
+              )
+            ];
         }
-
+        /** @type {valueParser.Dimension | null | false} */
         let lastStop = null;
```

Comment:
`lastStop` is going to contain the result of `valueParser.unit()`, which is either `Dimension` or `false`. Not sure whether this `null` is intentional or not because the same pattern is repeated elsewhere but with `lastStop` initialized to `false`. But in this case there's an explicit `if (lastStop === null)` instead of `(!lastStop)`.